__tests__/app.test.js

```diff
@@ -44,6 +40,41 @@ describe("GET /api/topics", () => {
     })
 })
 
+describe("GET /api/articles", () => {
+    test("200: returns array of all articles objects with the correct key values - including the key of body removed, and an added key of comment_count with the correct values. Array should be sorted by time created in descending order", () => {
+        return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then(({ body: { articles } }) => {
+                // exampleArticle manually counted as 11
+                const exampleArticle = articles.find(
+                    (article) =>
+                        article.title === "Living in the shadow of a great man"
```

Comment:
We're not really interested in specific articles for this test. As the endpoint is /api/articles, we just want to ensure we're getting all of the articles back and that the objects are in the correct structure